docs/src/test/scala/docs/http/scaladsl/Http2Spec.scala

```diff
@@ -50,4 +52,13 @@ object Http2Spec {
     Http().newServerAt("localhost", 8080).bind(handler)
     //#bindAndHandlePlain
   }
+
+  {
+    //#http2Client
+    Http().connectionTo("localhost").toPort(8080).http2()
```

Comment:
Maybe use a port that suggests this is a secure connection?

```suggestion
    Http().connectionTo("localhost").toPort(4433).http2()
```